src/web/metrics.rs

```diff
@@ -2,9 +2,12 @@ use super::pool::Pool;
 use iron::headers::ContentType;
 use iron::prelude::*;
 use iron::status::Status;
-use prometheus::{Encoder, IntGauge, IntCounter, TextEncoder};
+use prometheus::{
+    opts, register_counter, register_int_counter, register_int_gauge, Encoder, IntCounter,
+    IntGauge, TextEncoder, __register_gauge,
```

Comment:
What are these imports doing? `register_int_counter` and `register_int_gauge` are being used but the others don't look necessary.